include/session/config/contacts.hpp

```diff
@@ -97,6 +115,47 @@ struct contact_info {
     void load(const dict& info_dict);
 };
 
+struct blinded_contact_info : community {
+    using community::community;
```

Comment:
Inheriting from community here feels a little strange, because a blinded contact does not satisfy "is a community" but rather "has a community", and so I think you could do everything here more cleanly by just having a `community comm;` member instead of inheritance.

Reply:
So the main reason for the inheritance was so I could reuse the `comm_iterator_helper` - I've updated the struct to have a `community comm;` instead and added the `set_{x}` functions to redirect to setting the values on the member (not sure if this will work as desired or not 😬)